server/http/interceptors/interceptors.go

```diff
@@ -313,3 +314,13 @@ func WrapAuthenticatedExternalHandler(env environment.Env, next http.Handler) ht
 		RecoverAndAlert,
 	})
 }
+
+type HandlerFunc func(http.ResponseWriter, *http.Request) error
+
+func (f HandlerFunc) ServeHTTP(w http.ResponseWriter, r *http.Request) {
```

Comment:
nit: maybe rename `HandlerFunc` to `RedirectOnError`

so the call site is like

```
interceptors.RedirectOnError(auth.Login)
```

Reply:
Done